Resources/doc/dynamic.md

```diff
@@ -42,6 +42,8 @@ https://github.com/symfony/symfony/blob/v2.7.0/src/Symfony/Bridge/Twig/Resources
 </html>
 ```
 
+Attention: If used with a template.xml the form variable is accessed with content.form.
```

Comment:
@TschannettM would just remove this line and change it in the example to `form_theme content.form` and `form(content.form)`.